src/main/groovy/nebula/plugin/publishing/maven/MavenResolvedDependenciesPlugin.groovy

```diff
@@ -39,14 +40,19 @@ class MavenResolvedDependenciesPlugin implements Plugin<Project> {
                             def dependencies = xml.asNode()?.dependencies?.dependency
                             def dependencyMap = [:]
 
-                            dependencyMap['runtime'] = project.configurations.runtime.incoming.resolutionResult.allDependencies
+                            dependencyMap['runtime'] = project
+                                    .sourceSets.main.runtimeClasspath.sources
```

Comment:
yea that's an oversight on my part, switched to directly using the configuration